### DISCUSSION TITLE
Welcome to hellodata-be Discussions!

### DISCUSSION THREAD

**Discussion by Nicolassteiner** [kanton-bern/hellodata-be#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sspaeti** [comment]
Welcome everyone. Please let us know if you have any questions or want to discuss anything. We're super happy to get feedback :)

**Reply by saketbhat** [reply]
Ran the docker compose to pull images and run all the containers but the portal does not show anything except some hyberlinks . Running on a x86 intel mac

**Reply by nschmid** [reply]
Can you try to run/build with the `--build` flag?
e.g.:
`docker-compose pull;  docker-compose up -d --build`
... to fetch and build from scratch? Does that help?

**Reply by saketbhat** [reply]
Did not help

**Reply by sspaeti** [reply]
Hey @saketbhat, we couldn't try an Intel Mac, but I have an Apple silicon M1 chip. What worked for me is downloading the latest [Docker Desktop](https://www.docker.com/products/docker-desktop/) (it didn't work with the older version) and activating intel emulation (which you won't need on intel chip). Second, add `127.0.0.1 host.docker.internal` to the `/etc/hosts` according to Readme, which I guess you have already. After that, a `docker-compose down --volumes --remove-orphans` and `docker-compose pull;  docker-compose up -d --build` again.

If not working, what errors or messages do you have? Maybe you can also check the Postgres pod, sometimes, if you have not enough RAM/CPU for Docker Desktop, postgres struggles to process all requests. Let us know how that goes.
